From ebfff6a5e512c92c436a70d91772413534500c89 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Uwe=20Kleine-K=C3=B6nig=20=28The=20Capable=20Hub=29?=
 <u.kleine-koenig@baylibre.com>
Date: Fri, 3 Jul 2026 17:46:25 +0200
Subject: [PATCH 13281/15333] nfc: Unify style of spi_device_id arrays
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Unify the style of the list terminator in spi_device_id arrays, that
is use a single space between { and }. This is the most common and
generally recommended style for these.

Signed-off-by: Uwe Kleine-König (The Capable Hub) <u.kleine-koenig@baylibre.com>
Link: https://patch.msgid.link/78d632098fd42dbf2846cb89d66ec83bb9e1de99.1783091699.git.u.kleine-koenig@baylibre.com
Signed-off-by: David Heidelberg <david@ixit.cz>
---
 drivers/nfc/st-nci/spi.c  | 2 +-
 drivers/nfc/st95hf/core.c | 2 +-
 drivers/nfc/trf7970a.c    | 3 +--
 3 files changed, 3 insertions(+), 4 deletions(-)

diff --git a/drivers/nfc/st-nci/spi.c b/drivers/nfc/st-nci/spi.c
index 1b97b2f3f441..7948c7e0c88c 100644
--- a/drivers/nfc/st-nci/spi.c
+++ b/drivers/nfc/st-nci/spi.c
@@ -273,7 +273,7 @@ static void st_nci_spi_remove(struct spi_device *dev)
 static struct spi_device_id st_nci_spi_id_table[] = {
 	{ .name = ST_NCI_SPI_DRIVER_NAME },
 	{ .name = "st21nfcb-spi" },
-	{}
+	{ }
 };
 MODULE_DEVICE_TABLE(spi, st_nci_spi_id_table);
 
diff --git a/drivers/nfc/st95hf/core.c b/drivers/nfc/st95hf/core.c
index d4e3049d138a..321fbe8aeca8 100644
--- a/drivers/nfc/st95hf/core.c
+++ b/drivers/nfc/st95hf/core.c
@@ -1050,7 +1050,7 @@ static const struct nfc_digital_ops st95hf_nfc_digital_ops = {
 
 static const struct spi_device_id st95hf_id[] = {
 	{ .name = "st95hf" },
-	{}
+	{ }
 };
 MODULE_DEVICE_TABLE(spi, st95hf_id);
 
diff --git a/drivers/nfc/trf7970a.c b/drivers/nfc/trf7970a.c
index b9ea2b61c588..60883001fa5d 100644
--- a/drivers/nfc/trf7970a.c
+++ b/drivers/nfc/trf7970a.c
@@ -2311,9 +2311,8 @@ MODULE_DEVICE_TABLE(of, trf7970a_of_match);
 
 static const struct spi_device_id trf7970a_id_table[] = {
 	{ .name = "trf7970a" },
-	{}
+	{ }
 };
-
 MODULE_DEVICE_TABLE(spi, trf7970a_id_table);
 
 static struct spi_driver trf7970a_spi_driver = {
-- 
2.55.0

